.github/workflows/uml.yml

```diff
@@ -10,12 +10,10 @@ on:
     paths:
       - "pymc_marketing/**"
 
-permissions: write-all
-
 jobs:
   build:
     runs-on: ubuntu-latest
-
+    permissions: write-all
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
```

Comment:
Will try v4 instead

Also, make sure that the checked out hash is good to push and doesnt require additional pull